agent/lib/kontena/launchers/ipam_plugin.rb

```diff
@@ -34,12 +34,22 @@ def on_node_info(topic, info)
 
     def start(info)
       create_container(@image_name, info)
+      sleep 1 until running?
```

Comment:
Yes, there seems to be short window where the ipam container is running but not crashed yet. Maybe agent could poll the `/IpamDriver.GetCapabilities` endpoint?